### DISCUSSION TITLE
deep_learning/boosting_models_pytorch/weight_initialization_activation_functions/

### DISCUSSION THREAD

**Discussion by giscus** [ritchieng/deep-learning-wizard#25] [Website]
# deep_learning/boosting_models_pytorch/weight_initialization_activation_functions/

We try to make learning deep learning, deep bayesian learning, and deep reinforcement learning math and code easier. Used by thousands.

https://www.deeplearningwizard.com/deep_learning/boosting_models_pytorch/weight_initialization_activation_functions/

**Comment by LiviaCavalcanti** [comment]
I would like to clarify two points [in this section](https://www.deeplearningwizard.com/deep_learning/boosting_models_pytorch/weight_initialization_activation_functions/#tanh), it says that the tanh saturates at 0 and 1, and it has no signal to update weights. Is it really the case? the tanh has sign, it is its advantage in comparision to sigmoid. Also, its saturation is at -1 and 1 instead of 0 and 1, no?
